secretmanager/src/test/java/secretmanager/SnippetsIT.java

```diff
@@ -432,6 +457,16 @@ public void testCreateUpdateSecretLabel() throws IOException {
         UPDATED_LABEL_KEY, UPDATED_LABEL_VALUE);
   }
 
+  @Test
+  public void testEditSecretAnnotations() throws IOException {
+    SecretName name = SecretName.parse(TEST_SECRET.getName());
+    Secret updatedSecret = EditSecretAnnotations.editSecretAnnotations(
+        name.getProject(), name.getSecret(), UPDATED_ANNOTATION_KEY, UPDATED_ANNOTATION_VALUE);
+
+    assertThat(updatedSecret.getAnnotationsMap()).containsEntry(
+      UPDATED_ANNOTATION_KEY, UPDATED_ANNOTATION_VALUE);
```

Comment:
We can move this to BeforeClass annotation. That way the annotation is "added" to the existing TEST_SECRET.

Reply:
I am not sure what you mean here, why would we want move a test to BeforeClass?

Reply:
Few tests, like resource creation can be moved to BeforeClass so that the resource can be setup once and used by all the other tests in the class. This would avoid creating multiple resources for testing.

But I see you passed in the boolean to createSecret method. That works too.